pkg/apis/kubermatic/v1/constraint_template.go

```diff
@@ -41,13 +41,16 @@ type ConstraintTemplate struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
+	// Spec specifies the gatekeeper constraint template and kubermatic related spec.
```

Comment:
Same here. "kubermatic" is not the product name, it's a lowercased version of the company name. "KKP" would fit better.